src/Swarm/TUI/View/CellDisplay.hs

```diff
@@ -66,58 +80,75 @@ displayRobotCell g coords =
   map (view robotDisplay) $
     robotsAtLocation (fmap W.coordsToLoc coords) g
 
-displayEntityCell :: WorldEditor Name -> GameState -> Cosmic W.Coords -> [Display]
-displayEntityCell worldEditor g coords =
+mkEntityKnowledge :: GameState -> EntityKnowledgeDependencies
```

Comment:
Could we add some Haddock comments to `mkEntityKnowledge`, `EntityKnowledgeDependencies`, etc.?